dev/io.openliberty.checkpoint_fat_session_cache_infinispan/fat/src/io/openliberty/checkpoint/session/cache/infinispan/CheckpointSessionCacheOneServerTest.java

```diff
@@ -89,14 +92,17 @@ public static void setUp() throws Exception {
         options.put("-Dinfinispan.cluster.name", rand);
         options.put("-Dsession.cache.config.file", sessionCacheConfigFile);
         server.setJvmOptions(options);
-        server.setCheckpoint(CheckpointPhase.AFTER_APP_START, true, null);
+        server.setCheckpoint(CheckpointPhase.AFTER_APP_START, false, null);
         server.startServer();
+        configureEnvVariable(server, singletonMap("uri_location", "file:///${shared.resource.dir}/infinispan/infinispan-attr-meta-configuration.xml"));
```

Comment:
I am surprised variable substitution can be done in a value from the environment.  So this results in the final value for the URI to have filled in the shared resource dir?

Reply:
Yes, the final value has the shared resource dir location.